src/test/java/org/opensearch/security/sanity/tests/InvalidAdminPasswordIT.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ *
+ * Modifications Copyright OpenSearch Contributors. See
+ * GitHub history for details.
+ */
+
+package org.opensearch.security.sanity.tests;
+
+import org.hamcrest.MatcherAssert;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import org.opensearch.client.Request;
+import org.opensearch.client.Response;
+import org.opensearch.client.ResponseException;
+
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.is;
+
+public class InvalidAdminPasswordIT extends SecurityRestTestCase {
+
+    @BeforeClass
+    public static void setUpInvalidAdminCredentials() {
+        useAdminAsPassword = true;
+    }
+
+    @AfterClass
+    public static void restoreValidAdminCredentials() {
+        useAdminAsPassword = false;
+    }
```

Comment:
Why is this necessary? Does the useAdminAsPassword carry over to other files?

Reply:
this is to test that the request is not successful when the password is set to `admin`.